.github/workflows/lint.yml

```diff
@@ -27,7 +27,7 @@ jobs:
         composer config -g cache-dir "${{ env.COMPOSER_CACHE }}"
 
     - name: Prepare composer cache
-      uses: actions/cache@v3
+      uses: actions/cache@v2
```

Comment:
Is there a reason to degrade action cache to the prior version?

Reply:
> Is there a reason to degrade action cache to the prior version?

Copy paste error. I've removed it via an amend and force push.